python/tk_multi_publish2/thumbnail.py

```diff
@@ -113,9 +117,31 @@ def _on_screengrab(self):
             self._bundle.log_debug(
                 "Got screenshot %sx%s" % (pixmap.width(), pixmap.height())
             )
+            self._multiple_values = False
             self._set_screenshot_pixmap(pixmap)
             self.screen_grabbed.emit(pixmap)
 
+    def _set_multiple_values_indicator(self, is_multiple_values):
+        """
+        Specifies wether to show multiple values indicator
+        """
+        self._multiple_values = is_multiple_values
+
+    def paintEvent(self, paint_event):
+        """
+        Paint Event override
+        """
+        # paint thumbnail
+        QtGui.QLabel.paintEvent(self, paint_event)
+
+        # paint multiple values indicator
+        if self._multiple_values == True:
+            p = QtGui.QPainter(self)
+            p.setFont(QtGui.QFont("Arial", 22, QtGui.QFont.Bold))
+            pen = QtGui.QPen(QtGui.QColor("#FF0000"))
+            p.setPen(pen)
+            p.drawText(self.rect(),QtCore.Qt.AlignRight,"* ")
```

Comment:
May want to double check with @robblau on his opinion of a good way to represent multiple thumbnails. I'd suggest not showing a thumbnail and having text there. Maybe something like...

![screen shot 2017-12-15 at 1 52 14 pm](https://user-images.githubusercontent.com/2604646/34055993-7ab52f72-e19f-11e7-89a5-b28a74d86a7b.png)